### ISSUE TITLE
Phenotype values

### ISSUE THREAD

**Issue by ArianaHC** [statgen/EPACTS#31]
Hello everyone,

I have been trying to run a group-wise burden test, but I'm getting the same error over and over again:

`epacts group --vcf PPP_new_bial.vcf.gz --ped PPP.ped --max-maf 0.001 --groupf trunc.grp --pheno DISEASE --test skat --skat-o --out trunc.skat --run 2`
> ERROR: Phenotypes has only one or less unique values (2)

I have tried changing my ped file phenotype code from 1/2 to 0/1 and I've made sure that it's tab delimited. Does anyone have any idea how to fix this?

**Comment by jonathonl** [comment]
I'm guessing you need to replace `DISEASE` with the column name of your phenotype.

**Comment by ArianaHC** [comment]
> I'm guessing you need to replace `DISEASE` with the column name of your phenotype.

I have named the column for the phenotype `DISEASE`, but I've tried changing it to other things and it doesn't seem to solve the problem.

Thanks for the suggestion though.

**Comment by jonathonl** [comment]
Can you provide the first line of your ped file `head -n1 PPP.ped` and the full log output from your failed run? Does the vcf file and ped file contain the same set of samples?

**Comment by ArianaHC** [comment]
This is the first line of my ped file:
```
$ head -n 1 PPP.ped
#FID	IID	PID	MID	SEX	DISEASE
```
And the full log is in fact really short:

> ERROR: Phenotypes has only one or less unique values (2) at /app/software/EPACTS/3.3.2-foss-2016b/bin/epacts.pm line 276, <PED> line 1103.

There are more samples in the vcf file than in the ped file. Should I extract into another vcf file only the samples I want to analyse instead?

Thanks again for the help

**Comment by jonathonl** [comment]
No,  subsetting the VCF is not necessary. I'm not sure what's going on. Can you test with `--vcf data/1000G_exome_chr20_example_softFiltered.calls.vcf.gz --ped data/1000G_dummy_pheno.ped`? If it runs with the test data, then my only thought is to check for extra whitespace or carriage returns that could be affecting the parsing.

**Comment by ArianaHC** [comment]
It does indeed work with the test data, and I have looked for whitespaces and carriage returns and nothing there. 

Thanks for the suggestions though, it was worth trying
